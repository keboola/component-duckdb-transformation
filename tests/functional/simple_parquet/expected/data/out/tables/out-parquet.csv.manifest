{"destination": "out.c-out.out-parquet", "incremental": false, "write_always": false, "delimiter": ",", "enclosure": "\"", "manifest_type": "out", "has_header": true, "schema": [{"name": "id", "data_type": {"base": {"type": "INTEGER"}}, "nullable": true}, {"name": "value", "data_type": {"base": {"type": "STRING"}}, "nullable": true}]}