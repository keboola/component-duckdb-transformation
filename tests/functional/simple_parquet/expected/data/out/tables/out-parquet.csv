id,value
"1","foo"
"2","bar"
